pkg/serviceaccount/serviceaccounts_controller.go

```diff
@@ -44,8 +43,8 @@ func nameIndexFunc(obj interface{}) (string, error) {
 
 // ServiceAccountsControllerOptions contains options for running a ServiceAccountsController
 type ServiceAccountsControllerOptions struct {
-	// Names is the set of service account names to ensure exist in every namespace
-	Names util.StringSet
+	// ServiceAccounts is the list of service accounts to ensure exist in every namespace
+	ServiceAccounts []api.ServiceAccount
```

Comment:
Confused.  Since ServiceAccounts reference a Secret object, in a specific namespace, you are expecting that those are created before this namespace is created?